### PR TITLE
Lecture CI 

### PR DESCRIPTION
**Pull request by cocodrips**
## What
Add CI lecture

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: document/06-ci.ja.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## 2. Github Actionsを有効にする
+GithubはGithub Actionsと呼ばれるCIサービスを提供しています。
+まずは皆さんのリポジトリでGithub Actionを有効化します。
+
+- [ワークフローの有効化](https://docs.github.com/ja/actions/managing-workflow-runs/disabling-and-enabling-a-workflow)
+
+有効にしたら、適当なcommitを作成し、PushをトリガーにGithub Actionsのワークフローが動くことを確認します。
+以下の図のように:white_check_mark:や:red_circle:がcommit commentの横に表示されていれば動いています。
+![Enable workflow](../data/workflow-enable.png)
```
Comment (wakanapo) [inline document/06-ci.ja.md:24]: commitしわすれ？

Comment (cocodrips) [reply document/06-ci.ja.md:24]: ほんとだ

Comment (cocodrips) [reply document/06-ci.ja.md:24]: すみません更新しました！
[5b9fed2](https://github.com/mercari-build/mercari-build-training-2023/pull/4/commits/5b9fed22ac235b9120daf71032a4a77fb1577871)

**Review by wakanapo — APPROVED** [review]
🎉

**Pull request merged**